pkg/localdata/constant.go

```diff
@@ -67,9 +67,15 @@ const (
 	// EnvNameSSHPassPrompt is the variable name by which user specific the password prompt for sshpass
 	EnvNameSSHPassPrompt = "TIUP_SSHPASS_PROMPT"
 
-	// EnvNameNativeSSHClient is the variable name by which user can specific use natiive ssh client or not
+	// EnvNameNativeSSHClient is the variable name by which user can specific use native ssh client or not
 	EnvNameNativeSSHClient = "TIUP_NATIVE_SSH"
 
+	// EnvNameNativeSSHPath is the variable name by which user can specific the native executable ssh binary path
+	EnvNameNativeSSHPath = "TIUP_NATIVE_SSH_PATH"
```

Comment:
I think we could just use `TIUP_SSH_PATH`, or use `TIUP_SYSTEM_SSH_PATH` as we defines the former "native ssh" as "system ssh" for better understanding in #699, the former `TIUP_NATIVE_SSH` should, however, keep unchanged for backward compatibility.

Reply:
I prefer to `TIUP_SSH_PATH`, `TIUP_SCP_PATH`